pxr/imaging/plugin/hdRpr/package/readme.txt

```diff
@@ -0,0 +1 @@
+readme test
```

Comment:
? Is this intended?

Reply:
No, I don't think so, that was for testing purposes, I will remove it or find usage for it tomorrow